.gitignore

```diff
@@ -25,5 +26,3 @@ node_modules
 
 #other
 package-lock.json
-.editorconfig
-.eslintrc
```

Comment:
The settings on me winpc and mac are slightly different - so why bother to show these settings in the repository?